cfme/tests/automate/test_automate_manual.py

```diff
@@ -930,3 +930,127 @@ def test_git_refresh_with_renamed_yaml():
+            2. Navigate to Automation > Automate > Import/Export and upload test.zip file
+            3. Navigate to Automation > Automate > Customization > Import/Export and upload test.yml
+        expectedResults:
+            1.
+            2. Error message should be displayed
+            3. Error message should be displayed
+
+    Bugzilla:
+        1720611
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_initiate_retirement_automate_method():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Create a writeable domain
+            2. Create a /System/Request/retire_automation_service instance and set meth5 to
+               retire_automation_service.
+            3. Create a /System/Request/retire_automation_service method.
+               Sample code:
+               ```
+               service = $evm.root['service']
+               $evm.log(:info, "create_retire_request for  service #{service}")
+               request = $evm.execute(:create_retire_request, service)
+               $evm.log(:info, "Create request for create_retire_request for request #{request}")
+               ```
+            4. In Automation/Automate/Simulation:
+               >> Set Request => retire_automation_service
+               >> Select Object Attribute Type => Service
+               >> Selection => Service to be retired
+               and click on Submit
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Service should be retired from Automate method.
+
+    Bugzilla:
+        1700524
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_git_refresh_with_rapid_updates():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Have a git backed domain that imported cleanly (if possibly incompletely)
+            2. Break the domain in Git, or notice a method isn't visible because its matching .yaml
+               was never added to git
+            3. Add an broken .yaml to git, push, etc, in a desperate attempt to fix the issue.
+            4. Go to CF UI, Automate, Domain, "Refresh with a new branch or tag"
+            5. Select suitable branch and "Save"
+            6. Check evm.log
+            7. Fix e.g. <method>.yaml, commit, push
+            8. Refresh page you never left
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5. Error message should be displayed in UI
+            6. Errors should be available in logs
+            7.
+            8. It should re-pull or force user to do something if (5) is updated to block
+
+    Bugzilla:
+        1696396
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_simulation_copy_button():
```

Comment:
Isn't this automated by https://github.com/ManageIQ/integration_tests/pull/8938? @digitronik ?

Reply:
Nope this test case is different as it copy from simulation page and paste it in custom button. 
This(```test_simulation_copy_button```) test case just checking basic functionality of the copy button on simulation page respective to changes in the values on that page.